php/functions.php

```diff
@@ -471,3 +471,20 @@ function getProcessUptime($process)
     $secs = str_pad(floor($seconds % 60), 2, "0", STR_PAD_LEFT);
     return "$days days, $hours:$mins:$secs";
 }
+
+/**
+ * Gets the value of title_text from config if it exists
+ *
+ * @return string
+ */
+function getTitleText()
```

Comment:
This function is relatively superfluous, the template should echo the config value directly, to keep in-style with the rest of the code.

Admittedly, echoing values in-line into HTML isn't the _best_ way to do this, but in the interests of simplicity and lightness it's nicer than relying on Twig.